src/placeos/api_wrapper/system_triggers.cr

```diff
@@ -2,7 +2,55 @@ require "./endpoint"
 
 module PlaceOS
   class Client::APIWrapper::SystemTriggers < Client::APIWrapper::Endpoint
```

Comment:
@caspiano or @dukeraphaelng
is there a semantic difference between `trigger_instances` in src/models and `system_triggers` here ?